components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -1,34 +1,56 @@
-// legacy_hash_id: a_poiwMj
-import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+import { URL } from "url";
+import calendly from "../../calendly_v2.app.mjs";
 
 export default {
   key: "calendly_v2-get-event",
   name: "Get Event",
-  description: "Gets information about an Event associated with a URI.",
-  version: "0.1.1",
+  description: "Gets information about an Event associated with a URI. [See docs here](https://developer.calendly.com/api-docs/e2f95ebd44914-get-event).",
+  version: "0.1.2",
   type: "action",
   props: {
-    calendly_v2: {
-      type: "app",
-      app: "calendly_v2",
+    calendly,
+    eventId: {
+      propDefinition: [
+        calendly,
+        "eventId",
+      ],
+      optional: true,
     },
-    event_uuid: {
+    eventUrl: {
       type: "string",
-      description: "The event's unique identifier.",
+      label: "Event URL",
+      description: "The URL of the event to retrieve information about. If you are using a Calendly Source in the same workflow, you would use ``{{steps.trigger.event.payload.event}}``.",
+      optional: true,
+    },
+  },
```

Comment:
```suggestion
      description: "The URL of the event to retrieve information about. If you are using a Calendly Source in the same workflow, you would use {{steps.trigger.event.payload.event}}."
```

Reply:
Changed * to {{}} for consistency with Google's Developer docs style guide

Reply:
To confirm @dylburger we want to have {{}} but without the backticks ``?

Reply:
Nah, that's a bad suggestion from PR review bot. Let's leave them when wrapping code references like that.